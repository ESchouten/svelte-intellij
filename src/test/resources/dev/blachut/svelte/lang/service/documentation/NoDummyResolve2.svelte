<script lang="ts">
  let unrelatedData = {
    unrelatedValue: 42
  };

  let unrelatedData2 = {
    unrelatedValue: true
  };

  export let usedData: any;

  console.log(unrelatedData, unrelatedData2);
</script>

{usedData.<caret>unrelatedValue}
