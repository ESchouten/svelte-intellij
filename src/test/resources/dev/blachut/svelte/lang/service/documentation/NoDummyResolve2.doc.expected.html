<div class='definition'><pre>unrelatedData.unrelatedValue: <span style="color:#0000ff;">42&#32;</span><span style="">|&#32;</span><span style="color:#000080;">true</span></pre></div><table class='sections'><tr><td valign='top'><icon src='AllIcons.FileTypes.JavaScript'/>&nbsp;NoDummyResolve2.svelte</td></table><div class='content'><p><a href="psi_element:///private/var/folders/r0/xc0f60bj4j3bq1cjt8rf4kj80000gn/T/unitTest_noDummyResolve2_2RRIJyHXaItr5yLmIibX2atd3du/unitTest10718127077762439669/NoDummyResolve2.svelte%unrelatedValue%unrelatedData2:1">`unrelatedData2.unrelatedValue`</a></div>